spec/models/volunteer_op_spec.rb

```diff
@@ -310,4 +310,19 @@
       expect(VolunteerOp.get_source([l_vol_op1, d_vol_op2])).to eq('mixed')
     end
   end
+
+  describe 'a new volunteer opportunity' do
+    context 'is new volunteer opportunity' do
+      it 'returns true' do
+        new_volop = build(:volunteer_op, created_at: Time.now)
+        expect(new_volop.new_volop?).to be_truthy
```

Comment:
if the method was `new` here we could use rspec syntax like so:

```rb
expect(new_volop).to be_new
```

Reply:
Definitely, I like that! - I will make the changes.